conf/base.config

```diff
@@ -12,6 +12,7 @@ vim: syntax=groovy
  */
 
 process {
+  container = { "docker://nfcore/exoseq:${workflow.revision ? workflow.revision : 'latest'}" }
```

Comment:
I think that in the more recent versions of Nextflow we no longer need the `docker://` prefix for singularity pulls?

Reply:
Also `nf-core` not `nfcore` now I think right?